azurerm/data_source_monitor_action_group_test.go

```diff
@@ -170,7 +220,41 @@ resource "azurerm_monitor_action_group" "test" {
   webhook_receiver {
     name        = "callmybackupapi"
     service_uri = "https://backup.example.com/warning"
+	}
+
+	automation_runbook_receiver {
+		name = "action_name_1"
+		automation_account_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001"
+		runbook_name = "my runbook"
+		webhook_resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001/webhooks/webhook_alert"
+		is_global_runbook = true
+		service_uri = "https://s13events.azure-automation.net/webhooks?token=randomtoken"
+	}
+
+	voice_receiver {
+    name         = "oncallmsg"
+    country_code = "1"
+    phone_number = "1231231234"
   }
+
+  voice_receiver {
+    name         = "remotesupport"
+    country_code = "86"
+    phone_number = "13888888888"
+	}
+
+	logic_app_receiver {
+		name = "logicappaction"
+		resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg-logicapp/providers/Microsoft.Logic/workflows/logicapp"
```

Comment:
could we provision [a logic app workflow](https://www.terraform.io/docs/providers/azurerm/r/logic_app_workflow.html) dynamically too and reference it's ID here?